README.md

```diff
@@ -45,120 +61,103 @@ flowchart LR;
     S --> GS
     S --> CPS
     S --> CWS
-    S --> RS
     subgraph Public
         F(Frequency Network)
     end
     AS --> F
     GS --> F
     CPS --> F
     CWS --> F
-    RS --> F
 ```
 
 <p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
 
-## 🛠 Built With
-
-Each Gateway service is an independent microservice.
-
-### Tech Stack
-
-<details>
-<summary>Account Service</summary>
-
-- [API Documentation](https://projectlibertylabs.github.io/gateway/account)
-- [README](./developer-docs/account/README.md)
-
-</details>
-
-<details>
-<summary>Graph Service</summary>
-
-- [API Documentation](https://projectlibertylabs.github.io/gateway/graph/)
-- [README](./developer-docs/graph/README.md)
-
-</details>
-
-<details>
-<summary>Content Publishing Service</summary>
+## Gateway Microservices <a name="gateway-microservices"></a>
 
-- [API Documentation](https://projectlibertylabs.github.io/gateway/content-publishing/)
-- [README](./developer-docs/content-publishing/README.md)
+Gateway consists of four independent microservices, each designed to handle specific aspects of interaction with the Frequency blockchain. Below is a detailed overview of each service:
 
-</details>
+| Service | Description | API Documentation | README |
+|---------|-------------|-------------------|--------|
+| Account Service | Manages user accounts and authentication on the Frequency blockchain. It handles tasks such as account creation and key management. | [API Docs](https://projectlibertylabs.github.io/gateway/account) | [README](./developer-docs/account/README.md) |
+| Graph Service | Manages social connections and relationships between users on the Frequency network. It handles operations like following/unfollowing users and retrieving social graphs. | [API Docs](https://projectlibertylabs.github.io/gateway/graph/) | [README](./developer-docs/graph/README.md) |
+| Content Publishing Service | Facilitates the creation and publication of content on the Frequency blockchain. It manages tasks such as posting messages, creating threads, and content moderation. | [API Docs](https://projectlibertylabs.github.io/gateway/content-publishing/) | [README](./developer-docs/content-publishing/README.md) |
```

Comment:
nit: I feel like content moderation may need to be its own service and since we don't really have any of that functionality yet, maybe it doesn't belong here.
```suggestion
| Content Publishing Service | Facilitates the creation and publication of content on the Frequency blockchain. It manages tasks such as posting messages, attachments, replies, and reactions. | [API Docs](https://projectlibertylabs.github.io/gateway/content-publishing/) | [README](./developer-docs/content-publishing/README.md) |
```